src/app-components/readme.md

```diff
@@ -0,0 +1,11 @@
+## IMPORTANT
+
+Components in this folder should be 'dumb', meaning the should have no knowledge about the containing app.
+
+They should implement three things:
+
+1. Receive and display data in the form of props.
+2. Report data changes in a callback.
+3. Receive and display the validity of the data.
+
+These components will form the basis of our future component library, and will enable refactoring of the frontend app.
```

Comment:
Not sure that's how it's meant to be used, but might as well link to the ADR from here as well. :pray:

Reply:
This is more a general guideline for the components in the folder. My preference would be to have this lib completely separate from the src folder so that importing the wrong way is not possible, but as we have not decided to do this yet, this readme serves as a litt reminder :)